data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v5.1.xml

```diff
@@ -3,18 +3,106 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
(It is not related to this change.)
But the `creation` in the file name is not really accurate.
Maybe `migration`?
(But that can be done separately if needed at all.)